src/commcare_cloud/ansible/roles/pg_backup/tasks/backup_plain_clean.yml

```diff
@@ -1,3 +1,15 @@
+- name: Remove old Hourly Cron job
+  become: true
+  cron:
+    name: "Backup postgres hourly"
+    job: "/usr/local/sbin/create_postgres_dump.sh hourly {{ postgresql_backup_hours * 60 }}"
+    minute: 0
+    hour: "{{ nadir_hour_frequency|default(3) }}"
```

Comment:
should these `hour` and `minute` values line up with those in `pg_backup/tasks/backup_plain.yml `?